docs/ci-cd-environment/macos-xcode-15-image.md

```diff
@@ -4,6 +4,8 @@ Description: The macos-xcode15 image is a customized image based on MacOS 14.1,
 
 # macOS Sonoma Xcode 15 image
 
+!!! warning "Xcode 15 image is currently in the Technical Preview stage. If you're interested in trying them out, please contact our support team."
```

Comment:
I'd rather say "Note:" than "!!! warning"

Reply:
Ivana is right.

Reply:
Yes. The word _warning_ won't show